### PR TITLE
Added "Center Linear " option to linear mode in Distribute Matrix node

### PR DESCRIPTION
**Pull request by kdsin23**
center linear

### CODE REVIEW

**Comment by Gerstmann-Bradley** [comment]
If we were to improve the function of linear mode, I wonder if it's possible to decide which direction this linear will go towards. like X, Y, Z.  I mostly use grid mode instead of linear mode because of this reason.
And Center Linear option is great. Thanks!

**Comment by OmarEmaraDev** [comment]
Feel free to implement @Gerstmann-Bradley's suggestions.

File: animation_nodes/nodes/matrix/distribute_matrices.pyx [hunk 1]
```diff
@@ -137,6 +137,8 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
         col.prop(self, "mode", text = "")
         if self.mode in ("LINEAR", "GRID"):
             col.prop(self, "distanceMode", text = "")
+            if self.mode == "LINEAR":
+                layout.prop(self, "centerAlongX", text = "Center Linear", toggle = True)
```
Comment (OmarEmaraDev) [inline animation_nodes/nodes/matrix/distribute_matrices.pyx:141]: If we will be sharing those properties between modes, make the property description more general above.

**Review by OmarEmaraDev — CHANGES_REQUESTED** [review]

File: animation_nodes/nodes/matrix/distribute_matrices.pyx (hunk 1 above)
Comment (kdsin23) [reply animation_nodes/nodes/matrix/distribute_matrices.pyx:141]: linear mode

is this OK?

Comment (Gerstmann-Bradley) [reply animation_nodes/nodes/matrix/distribute_matrices.pyx]: looks good to me at least. Thanks for development.

**Review by OmarEmaraDev — APPROVED** [review]
Thanks!

**Pull request merged**